common/matchmaking.ts

```diff
@@ -19,12 +20,12 @@ export enum MatchmakingType {
 
 export const ALL_MATCHMAKING_TYPES: ReadonlyArray<MatchmakingType> = Object.values(MatchmakingType)
 
-export function matchmakingTypeToLabel(type: MatchmakingType): string {
+export function matchmakingTypeToLabel(type: MatchmakingType, t?: TFunction): string {
```

Comment:
I know we needed this to be optional for team names, but I don't think this one needs to be? If you're just doing that for migration purposes you should leave a TODO + file an issue for cleaning it up